assignments/array-methods.js

```diff
@@ -54,30 +54,46 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
 let fullName = [];
+runners.foreach(` ${ runners[i]["first_name"] } ${ runners[i]["last_name"] }`);
+
+runners.foreach(fullName.push(runners[i].first_name + " " + runners[i].last_name ));
```

Comment:
The inside you made `fullName.push(runners[i].first_name + " " + runners[i].last_name` is correct if you were trying to do for loops(we weren't trying to do for loops for this assignment though), you just would have to had put it in a for loop like this `for (let i = 0; i < runners.length;i++){
  fullName.push(runners[i].first_name + " " + runners[i].last_name )
}`